### DISCUSSION TITLE
Suport  Brazilian Portuguese

### DISCUSSION THREAD

**Discussion by Raff-117** [Romanitho/Winget-AutoUpdate#462] [Ideas]
I did the translation from the French language version 1.18 to support my language. I don't know how to send the file, so I put it on onedrive.

https://senacpernambuco-my.sharepoint.com/:f:/g/personal/rafaellima_pe_senac_br/EmamHcLbeJBOtuE5kVWICScBLns45knc2AgRqUbG08hoWA?e=AB7zZC

**Comment by Romanitho** [comment]
To "send" the file, you need to fork the project, upload the file in your fork, and propose a pull request to our repository 👌🏼
Moreover, you'll automatically be credited.
It must be called pt.xml.

**Reply by Raff-117** [reply]
I understand. But, the Portuguese language of Portugal is pt-PT and Brazilian Portuguese is pt-BR.

**Reply by Romanitho** [reply]
Are there any differences between the two languages for these little notifications? Because we use the parent country code to detect the language to use.

**Reply by Raff-117** [reply]
Yes, there are some differences in language that make comprehension awkward in some texts. I can't say about the notifications, but if someone contributes to the Portuguese of Portugal he will point out the differences. So it's okay to just keep "pt" in the moment.

**Reply by Romanitho** [reply]
I understand. That's why in a first place we decided to use "generic" languages, to catch maximum users. If in the future things become more and more accurate, we will add specific language of countries :)

**Reply by Raff-117** [reply]
Perfect!
